diffsync/helpers.py

```diff
@@ -350,11 +350,7 @@ def sync_diff_element(self, element: DiffElement, parent_model: Optional["DiffSy
         attrs = diffs.get("+", {})
 
         # Retrieve Source Object to get its flags
-        src_model: Optional["DiffSyncModel"]
-        try:
-            src_model = self.src_diffsync.get(self.model_class, ids)
-        except ObjectNotFound:
-            src_model = None
+        src_model = self.src_diffsync.get_or_none(self.model_class, ids)
```

Comment:
Why is this change here? Pylint was complaining about too many branches and this method was recently introduced and fits this pattern exactly.